### ISSUE TITLE
`program.command(...).description(...).requiredOption` is not a function

### ISSUE THREAD

**Issue by bartenra** [pixari/vue-i18n-extract#125]
When I run `yarn run vue-i18n-extract` without any arguments. Not sure what's happening here.

```
/home/bart/src/enra_v3/frontend/node_modules/vue-i18n-extract/bin/vue-i18n-extract.js:23
  .requiredOption(
   ^

TypeError: program.command(...).description(...).requiredOption is not a function
    at Object.<anonymous> (/home/bart/src/enra_v3/frontend/node_modules/vue-i18n-extract/bin/vue-i18n-extract.js:23:4)
    at Module._compile (node:internal/modules/cjs/loader:1092:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1121:10)
    at Module.load (node:internal/modules/cjs/loader:972:32)
    at Function.Module._load (node:internal/modules/cjs/loader:813:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47
```

Node v15.14.0

Edit: I also get this error with arguments.

**Comment by bartenra** [comment]
When I run it with the `-v` and `-l` arguments it just prints `7.10.0` and then exists.

**Comment by pixari** [comment]
Hi @bartenra!
Do you still experience this problem?

Can you provide some information to reproduce it?

**Comment by Spittal** [comment]
Solved in v2.0.0 and above, There are now better explanations for required options.
